app/src/main/java/com/github/kiolk/cowsandbulls/utils/ThemeHelper.java

```diff
@@ -0,0 +1,35 @@
+package com.github.kiolk.cowsandbulls.utils;
+
+import android.os.Build;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatDelegate;
+
+public class ThemeHelper {
+
+    public static final String LIGHT_MODE = "light";
```

Comment:
I think, in storage setting repository we can save not string a use int, because modes a int constant. How result, reduced code in this class.